### ISSUE TITLE
Logitech Gaming Hub [Logitech.LGH] - Installer hash does not match

### ISSUE THREAD

**Issue by hexadezi** [microsoft/winget-pkgs#9185]
Create issues related to the winget.exe client [here](https://github.com/microsoft/winget-cli/issues/new?template=Bug_Report.md)

# Brief description of your issue

When installing, I get "Installer hash does not match error".
# Steps to reproduce

winget install -e --id Logitech.LGH

# Expected behavior

A successful install.

# Actual behavior

Gefunden Logitech Gaming Hub [Logitech.LGH]
Diese Anwendung wird von ihrem Besitzer an Sie lizenziert.
Microsoft ist nicht verantwortlich und erteilt keine Lizenzen für Pakete von Drittanbietern.
Downloading https://download01.logi.com/web/ftp/pub/techsupport/gaming/lghub_installer.exe
  ██████████████████████████████  39.4 MB / 39.4 MB
Installer-Hash stimmt nicht überein; zum Aufheben der Überprüfung --force

# Environment

```none
[winget --info]
Windows Package Manager v0.2.10191 Vorschau
Windows: Windows.Desktop v10.0.19042.870
Paket: Microsoft.DesktopAppInstaller v1.11.10191.0

Any other software?
```

**Comment by denelon** [comment]
@labo89 it looks like the publisher changed the installer. Our automation generally runs daily and should catch this and create a PR with the correct hash. It may not run depending on where we are with manifest conversation. Optionally, someone may download the installer, calculate the hash `winget hash lghub_installer.exe`, update the manifest and submit a PR.

**Comment by jedieaston** [comment]
The problem is that the Logitech Gaming Hub won't install silently anymore, so we can't update it in the repo. Maybe it should be removed.

See #7901 (among others if you do a search)

**Comment by denelon** [comment]
@jedieaston, thanks for sharing your knowledge on this. You may be right. If there isn't a silent installer available, we may be blocked here. The new manifest specification includes the ability to identify that a package only supports an interactive installer. We're looking at adding a feature to identify packages that can't install silently and filter those from search with some kind of arguments. Some community feedback on this might help prioritize that work. What are your thoughts on the subject? I'll create a discussion item, and link it.

**Comment by denelon** [comment]
https://github.com/microsoft/winget-pkgs/discussions/9190
Should the catalog allow "non-silent" installers?

**Comment by Arseni159** [comment]
I think we should keep the non-silent installers for the simple reason that even if it doesn't install completely automatically it's still a lot handier than going to their website and downloading the installer.

**Comment by Teraskull** [comment]
Another example is [FreeFileSync](https://freefilesync.org/). The silent install is only available with donation edition, but it is still possible to install the software in interactive mode with Winget.

**Comment by denelon** [comment]
https://github.com/microsoft/winget-cli/issues/910

**Comment by Trenly** [comment]
I think this can be closed because of @ItzLevvie's PR #15917, which removed the package since it couldn't be updated
